### PR TITLE
flake8, isort, black, and sphinx in CI

### PR DESCRIPTION
**Pull request by crusaderky**
Run in CI:
- flake8
- isort
- black
- sphinx-build

### CODE REVIEW

**Comment by crusaderky** [comment]
@jonathanslenders ready for review and merge

**Comment by crusaderky** [comment]
@jonathanslenders ping

File: examples/benchmarks/speed_test.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env python
-"""
-Benchmank how long it takes to set 10,000 keys in the database.
+"""Benchmank how long it takes to set 10,000 keys in the database.
```
Comment (crusaderky) [inline examples/benchmarks/speed_test.py]:
```suggestion
"""Benchmark how long it takes to set 10,000 keys in the database.
```

**Comment by jonathanslenders** [comment]
Thanks @crusaderky! Could you squash the commits?

**Comment by crusaderky** [comment]
@jonathanslenders done (FYI there's a button in the github PR merge for that)

**Comment by jonathanslenders** [comment]
Right. I forgot we can now do it from within GitHub. Thanks for this. I appreciate it!

**Pull request merged**